fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1,299 @@
+
```

Comment:
Just realised while building the gem works fine, the hidden files aren't packaged in general, so e.g. [.gitignore](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/plugins/template/.gitignore) isn't included in 1.93.1 — this feels like a separate issue, though. Do you think it makes sense to tackle that before continuing with this PR?